### PR TITLE
Wire up for centralized publish repo

### PR DESCRIPTION
**Pull request by chadwhitacre**
Copy pasta from [`snuba-sdk`](https://github.com/getsentry/snuba-sdk) (🎩 @BYK)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: build
+
+on:
+  push:
+    branches:
+      - release/**
```
Comment (BYK) [inline .github/workflows/build.yml:6]: Why not `master` too for "nightly" builds?

File: scripts/bump-version.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+echo "Current version: $OLD_VERSION"
+echo "Bumping version: $NEW_VERSION"
+
+function replace() {
+    ! grep "$2" $3
+    perl -i -pe "s/$1/$2/g" $3
+    grep "$2" $3  # verify that replacement was successful
+}
+
+replace "version=\"[0-9.]+\"" "version=\"$NEW_VERSION\"" ./setup.py
```
Comment (BYK) [inline scripts/bump-version.sh:24]: Version string may include dashes and letters too sometimes?

**Review by BYK — APPROVED** [review]

File: scripts/bump-version.sh (hunk 2 above)
Comment (chadwhitacre) [reply scripts/bump-version.sh:24]: Relaxed in 64c0104cd6f417acebbfd9819098355142f39b86. 👍

File: scripts/bump-version.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+OLD_VERSION="${1}"
+NEW_VERSION="${2}"
+
+echo "Current version: $OLD_VERSION"
+echo "Bumping version: $NEW_VERSION"
+
+function replace() {
+    ! grep "$2" $3
+    perl -i -pe "s/$1/$2/g" $3
```
[the remark this answers is not in the document]
Comment (chadwhitacre) [reply scripts/bump-version.sh:20]: Saved in 64c0104cd6f417acebbfd9819098355142f39b86. 😁 👍

File: .github/workflows/build.yml (hunk 1 above)
Comment (chadwhitacre) [reply .github/workflows/build.yml:6]: Not worth the trouble right now, this repo gets hardly any traffic.

**Pull request merged**